core/src/storage/impls/state.rs

```diff
@@ -262,9 +262,11 @@ impl<'a> StateTrait for State<'a> {
         &mut self, access_key_prefix: StorageKey,
```

Comment:
In line 250, Could you also change .delete(), to set with value MptValue::<Box<[u8]>>::TOMBSTONE.unwrap()?